cmake/CompilerFlags.cmake

```diff
@@ -27,6 +27,7 @@ function(set_compiler_options PROJECT_NAME ENABLE_WERROR)
             -Werror=unused-variable
             -Wunused-function
             -Werror=unused-function
+            -Werror=unused-result  # Error out on unused results of functions marked with LLPC_NODISCARD
```

Comment:
LGTM - other than your commit message says "unused variable" rather than "unused result"

Reply:
Good catch, I updated the commit message. Thanks!